scalameta/testkit/src/test/scala/scala/meta/testkit/ScalametaParserProperties.scala

```diff
@@ -74,6 +77,11 @@ object ScalametaParserPropertyTest extends FunSuiteLike {
   import ScalametaParserProperties._
   def main(args: Array[String]): Unit = {
     val result = runAnalysis()
+    val mdResult = Observation.markdownTable(result)
+    Files.write(
+      Paths.get("target", "parser_bugs.md"),
+      mdResult.getBytes())
+    println(mdResult)
     val parserProken = result.count(_._2.kind == ParserBroken)
     val prettyPrinterBroken = result.count(_._2.kind == PrettyPrinterBroken)
     println(s"""Parser broken: $parserProken
```

Comment:
On the line below, I'm OK with bumping the number 9 to 11, if we add ignored minimized tests for the remaining errors.